test/e2e-tests.sh

```diff
@@ -85,6 +85,18 @@ toggle_feature autocreate-cluster-domain-claims true config-network || fail_test
 go_test_e2e -timeout=2m ./test/e2e/domainmapping ${E2E_TEST_FLAGS} || failed=1
 toggle_feature autocreate-cluster-domain-claims false config-network || fail_test
 
+toggle_feature dataplane-trust enabled config-network || fail_test
```

Comment:
As https://github.com/knative/serving/pull/14472 was merged, we need to update this.

Reply:
perfect, I'll do that today. Nice to see things coming together 😄